### DISCUSSION TITLE
Changing opacity of timeline_line:

### DISCUSSION THREAD

**Discussion by croak9854** [tomasklaen/uosc#583] [Q&A] [answered]
Can we change the opacity of just timeline_line just like timeline_chapters_opacity? 

It would be easier to see the progress if we can decrease the opacity of the timeline but not the timeline_line.

**Comment by tomasklaen** [comment, marked as answer]
Can't right now, but I added it to the issues. You can track it in #584.

**Reply by croak9854** [reply]
Thank you.
